### ISSUE TITLE
template literal support

### ISSUE THREAD

**Issue by mfulton26** [otovo/htmx-tags#6]
Various tools support JavaScript template literals prefixed with an "HTML" comment. VSCode provides syntax highlighting for the following and Prettier will format it too:

```js
const html = /* HTML */ ` <script src="https://unpkg.com/htmx.org@1.9.6"></script>
  <!-- have a button POST a click via AJAX -->
  <button hx-post="/clicked" hx-swap="outerHTML">Click Me</button>`;
```

I'd like to see htmx-tags support HTMX attributes inside JavaScript template literals too.

**Comment by erhanertem** [comment]
Indeed, its needed. Too bad that currently no tool in vscode marketplace supports that.
